CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## v0.180.3, 4 April 2022
+
+- Add support for Python packages with epoch version [#4928](https://github.com/dependabot/dependabot-core/pull/4928)
```

Comment:
We could consider this a new feature and do a `minor` bump, but we could also consider it a bugfix 🤷

Reply:
Yeah I was leaning towards patch but it's definitely arguable 🤷